### PR TITLE
RELATED: SD-976 dataSet is optional in KD

### PR DESCRIPTION
**Pull request by ptquang86**
# Related Jira tasks
- [SD-976](https://jira.intgdc.com/browse/SD-976)

### CODE REVIEW

**Comment by yenkins** [comment]
Currently destined for [release R210](https://confluence.intgdc.com/display/Dev/R210).
This release is **zero downtime** release.
Code drop: 2020-07-30 00:00.

**Comment by yenkins** [comment]
Story [SD-976](http://jira.intgdc.com/browse/SD-976) KD - Handling postMessage actively 
	Status: Hacking
	Priority: Major
	Assignee: Quang Pham

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/135/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/222/ : SUCCESS in 48s

File: src/embedding/common.ts [hunk 1]
```diff
@@ -224,16 +225,39 @@ export namespace EmbeddedGdc {
...
     export type AttributeFilterItem = IPositiveAttributeFilter | INegativeAttributeFilter;
     export type DateFilterItem = IAbsoluteDateFilter | IRelativeDateFilter;
     export type FilterItem = DateFilterItem | AttributeFilterItem;
     export type ObjQualifier = AFM.ObjQualifier;
 
-    export const isDateFilter = AFM.isDateFilter;
-    export const isRelativeDateFilter = AFM.isRelativeDateFilter;
-    export const isAbsoluteDateFilter = AFM.isAbsoluteDateFilter;
+    type ExtendedFilter = FilterItem | AFM.IMeasureValueFilter;
+    type CompatibilityFilter = AFM.IExpressionFilter | ExtendedFilter;
```
Comment (lvkhanh) [inline src/embedding/common.ts:251]: should be `type CompatibilityFilter = AFM.IExpressionFilter | FilterItem | AFM.IMeasureValueFilter;`

Comment (ptquang86) [reply src/embedding/common.ts]: updated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/136/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/223/ : SUCCESS in 32s

**Comment by lvkhanh** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/143/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/231/ : SUCCESS in 21s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/47/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/232/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-typings-fossa-scanning-zuul-el7/10/ : SUCCESS in 9s (non-voting)

**Pull request merged**